### PR TITLE
Content fixes

### PR DESCRIPTION
**Pull request by bwsw**

### CODE REVIEW

**Comment by bwsw** [comment]
@DaanHoogland, @rhtyd, @rafaelweingartner, @PaulAngus please review.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,15 @@ This is built using [middleman](https://middlemanapp.com)
     $ gem install middleman
     $ bundle install
 
-All sources are in the `/source` directory
+All sources are located in the `/source` directory. The build process requires JavaScript engine to be installed. For Ubuntu Linux use:
+
+```bash
+sudo apt-get update
+sudo apt install nodejs
```
Comment (rhtyd) [inline README.md]: You would only install middleman, no need of node.

**Comment by rhtyd** [comment]
Thanks @bwsw I'm on holiday next week, will review soon.

File: README.md (hunk 1 above)
Comment (bwsw) [reply README.md]: @rhtyd It doesn't work in Ubuntu without that.

**Comment by bwsw** [comment]
@rhtyd ping

File: content/developers.html [hunk 2]
```diff
@@ -267,17 +335,16 @@ <h3 class="panel-title">Git Repositories</h3>
 <ul>
 <li><a href="https://gitbox.apache.org/repos/asf/cloudstack.git" target="_blank">Apache CloudStack source code</a></li>
 <li><a href="https://gitbox.apache.org/repos/asf/cloudstack-cloudmonkey.git" target="_blank">Apache CloudStack Cloudmonkey source code</a></li>
-<li><a href="https://github.com/apache/cloudstack-ec2stack" target="_blank">Apache CloudStack EC2stack Inteface</a></li>
-<li><a href="https://github.com/apache/cloudstack-gcestack" target="_blank">Apache CloudStack GCEstack Interface</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs" target="_blank">General Documentation</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-install" target="_blank">Installation Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-admin" target="_blank">Administrative Guide</a></li>
-<li><a href="https://github.com/apache/cloudstack-docs-rn" target="_blank">Release Notes</a></li>
```
Comment (rhtyd) [inline content/developers.html]: Should we eventually deprecate and remove the link? What do you think @PaulAngus ?

Comment (PaulAngus) [reply content/developers.html]: yes eventually.  but 4.9 is still relevant to alot of people just now

**Pull request merged**